React/Views/RCTProgressView.h

```diff
@@ -17,8 +17,8 @@
 #endif
 
 #if TARGET_OS_OSX
-@property (nonatomic, strong, nullable) UIColor *progressTintColor;
-@property (nonatomic, strong, nullable) UIColor *trackTintColor;
+@property (nonatomic, strong, nullable) RCTUIColor *progressTintColor;
+@property (nonatomic, strong, nullable) RCTUIColor *trackTintColor;
```

Comment:
Missing the TODO comment

Reply:
Entire file is `// TODO(macOS ISS#2323203)`